### PR TITLE
Update QoS-in-Teams.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6279
Added tip.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/QoS-in-Teams.md [hunk 1]
```diff
@@ -177,6 +177,10 @@ new-NetQosPolicy -Name "Teams Sharing" -AppPathNameMatchCondition "Teams.exe" -I
 -IPSrcPortEndMatchCondition 50059 -DSCPAction 18 -NetworkProfile All
 ```
 
+
+> [!TIP]
+> The copy feature copies the text with a line return, which makes the command when pasted into a PowerShell terminal window fail. As a workaround, delete the line return to ensure the copy/paste feature works properly.
```
Comment (JohanFreelancer9) [inline Teams/QoS-in-Teams.md:182]:
```suggestion
> The copy feature copies the text with a line return, which makes the command fail when pasted into a PowerShell terminal window. As a workaround, delete the line return to ensure the copy/paste feature works properly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by SerdarSoysal** [comment]
@scanum 6279 can now be closed.

**Pull request merged**